<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalInformation</class>
 <widget class="QWidget" name="PersonalInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;</string>
   </property>
   <property name="text">
    <string>编辑个人信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>昵称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>简介：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>生日：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px;</string>
   </property>
   <property name="text">
    <string>地区：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nicknameLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>681</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="brief_introductionTextEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>681</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="maleRadioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>460</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="femaleRadioButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QComboBox" name="yearComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>2000</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2002</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2003</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2006</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2007</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2008</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2011</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2012</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2013</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2015</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2016</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2017</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2018</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2019</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2020</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2021</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2025</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="monthComboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dayComboBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>530</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>31</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cityComboBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>590</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <widget class="QComboBox" name="provinceComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>590</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>河北省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山西省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>辽宁省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>吉林省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑龙江省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江苏省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>浙江省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>安徽省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>福建省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>江西省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>山东省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>河南省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖北省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湖南省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广东省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>海南省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>四川省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>贵州省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>云南省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>陕西省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>甘肃省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>青海省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>台湾省</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>内蒙古自治区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>广西壮族自治区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>西藏自治区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>宁夏回族自治区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新疆维吾尔自治区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>北京市</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天津市</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上海市</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>重庆市</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>香港特别行政区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>澳门特别行政区</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>680</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>680</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>850</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 249, 252);</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>label_2</zorder>
  <zorder>nicknameLineEdit</zorder>
  <zorder>yearComboBox</zorder>
  <zorder>brief_introductionTextEdit</zorder>
  <zorder>maleRadioButton</zorder>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>saveButton</zorder>
  <zorder>femaleRadioButton</zorder>
  <zorder>monthComboBox</zorder>
  <zorder>label</zorder>
  <zorder>cityComboBox</zorder>
  <zorder>cancelButton</zorder>
  <zorder>provinceComboBox</zorder>
  <zorder>dayComboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
